src/Sulu/Component/Webspace/Manager/WebspaceManager.php

```diff
@@ -362,6 +370,17 @@ public function getWebspaceCollection(): WebspaceCollection
             require_once $cache->getPath();
 
             $this->webspaceCollection = new $class();
+
+            $currentRequest = $this->requestStack->getCurrentRequest();
+
+            // TODO change default to whatever is used for sitemaps, so the code can also be removed there
+            $host = $currentRequest ? $currentRequest->getHost() : '{host}';
```

Comment:
Can we if we have no request available fallback to the parameter which symfony uses to generate routes for CLI:

https://symfony.com/doc/current/routing.html#generating-urls-in-commands

Reply:
If I remember correclty I used that also for the dump of the sitemap.